src/main/java/AppointmentPage.java

```diff
@@ -40,37 +40,91 @@ public AppointmentPage(WebDriver driver) {
     private WebElement getTextLoginFailed;
 
-    public void clickDropDown(){
+    @FindBy(xpath = "//input[@id='txt_visit_date']")
+    private WebElement clickVisitDate;
+
+    @FindBy(xpath = "//td[normalize-space()='23']")
+    private WebElement enterVisitDate;
+
+    @FindBy(xpath = "//div[@class='datepicker-days']//th[@class='next'][normalize-space()='»']")
+    private WebElement nextMounth;
+
+    @FindBy(xpath = "//div[@class='datepicker-days']//th[@class='prev'][normalize-space()='«']")
+    private WebElement prevMounth;
+
+    @FindBy(xpath = "//textarea[@id='txt_comment']")
+    private WebElement clickCommentTextField;
+
+    @FindBy(xpath = "//button[@id='btn-book-appointment']")
+    private WebElement clickBookAppointment;
+
+
+    public AppointmentPage clickDropDown(){
         dropDownSelector.click();
+        return this;
     }
-    public void clickTokyoCuraHealtCareCenter(){
+    public AppointmentPage clickTokyoCuraHealtCareCenter(){
         tokyoCuraHealtCareCenter.click();
+        return this;
     }
-    public void clickHongkongCuraHealtCareCenter(){
+    public AppointmentPage clickHongkongCuraHealtCareCenter(){
         hongkongCuraHealtCareCenter.click();
+        return this;
     }
 
-    public void clickSeoulCuraHealtCareCenter(){
+    public AppointmentPage clickSeoulCuraHealtCareCenter(){
         seoulCuraHealtCareCenter.click();
+        return this;
     }
 
-    public void clickApplyForHospitalreadmission (){
+    public AppointmentPage clickApplyForHospitalreadmission (){
         applyForHospitalReadmission.click();
+        return this;
     }
-    public void clickHealtCareProgramMedicare(){
+    public AppointmentPage clickHealtCareProgramMedicare(){
         healtCareProgramMedicareRadioButton.click();
+        return this;
     }
-    public void clickHealtCareProgramMedicaid(){
+    public AppointmentPage clickHealtCareProgramMedicaid(){
         healtCareProgramMedicaidRadioButton.click();
+        return this;
     }
-    public void clickHealtCareProgramNone(){
+    public AppointmentPage clickHealtCareProgramNone(){
         healtCareProgramNoneRadioButton.click();
+        return this;
     }
     public String getTextMakeAppointment(){
         return makeAppointText.getText();
     }
     public String getTextLoginFailed(){
         return getTextLoginFailed.getText();
     }
+    public AppointmentPage clickVisitDate(){
+        clickVisitDate.click();
+        return this;
+    }
+    public AppointmentPage enterVisitDate(){
+        enterVisitDate.click();
+        return this;
+    }
+    public AppointmentPage nextMount(){
+        nextMounth.click();
+        return this;
+    }
+    public AppointmentPage prevMount(){
+        prevMounth.click();
+        return this;
+    }
+
+    public AppointmentPage clickCommentTF(String comment){
+        //clickCommentTextField.click();
+        clickCommentTextField.sendKeys(comment);
```

Comment:
Incorrect name of the method:  "clickCommentTF"
It should be putComment, or enterComment